doc/devel.md

```diff
@@ -0,0 +1,24 @@
+# Setup development environment
+
+This section contains some information that might be relevant for you when setting up a `corrosion2` development environment.  Depending on your computer setup your experience will vary.
```

Comment:
We should refer to Corrosion as just "corrosion". V2 is kind of internal to differentiate with the "proof of concept" that we had running before.